charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -212,6 +212,9 @@ spec:
             {{- range .Values.sidecars.provisioner.additionalArgs }}
             - {{ . }}
             {{- end }}
+            {{- if .Values.controller.enableMetrics }}
```

Comment:
Needs a guard against `.Values.controller` == `nil` (and this applies to every occurrence of this `if`)

Reply:
No longer applicable after deprecation of `--reuse-values`